src/audio/eq_iir/tune/example_fir_eq.m

```diff
@@ -10,13 +10,16 @@ function example_fir_eq()
 
 %% Common definitions
 fs = 48e3;
-fn.cpath3 = '../../ctl/ipc3/eq_fir';
-fn.cpath4 = '../../ctl/ipc4/eq_fir';
-fn.tpath1 =  '../../topology/topology1/m4';
-fn.tpath2 =  '../../topology/topology2/include/components/eqfir';
+sof_tools = '../../../../tools';
```

Comment:
This breaks bisect, so ideally should be a single commit. But given these are examples, ok in this case.

Reply:
I could combine first two commits. Would that be better?

Reply:
I've now combined commits 1 and 2.